pontoon/base/static/js/l20n.js

```diff
@@ -0,0 +1,2831 @@
+var L20n = (function () {
+  'use strict';
+
+  /*eslint no-magic-numbers: [0]*/
+
+  const locales2rules = {
```

Comment:
Ah, okay... I think we can discard this comment -> This is our copy of the l20n lib, Am I right?